crates/bevy_ui/src/lib.rs

```diff
@@ -82,9 +82,7 @@ impl Plugin for UiPlugin {
             // add these stages to front because these must run before transform update systems
             .add_system_to_stage(
                 CoreStage::PostUpdate,
-                widget::text_system
-                    .before(UiSystem::Flex)
-                    .after(ModifiesWindows),
+                widget::text_system.after(UiSystem::Flex),
```

Comment:
Needs to run after UiSystem::Flex but this doesn't seem like a safe solution, open to ideas